src/tools/opt-dist/src/main.rs

```diff
@@ -169,7 +159,7 @@ fn create_environment(args: Args) -> anyhow::Result<(Environment, Vec<String>)>
                 .shared_llvm(true)
                 // FIXME: Enable bolt for aarch64 once it's fixed upstream. Broken as of December 2024.
                 .use_bolt(!is_aarch64)
-                .skipped_tests(skip_tests)
+                .skipped_tests(vec![])
```

Comment:
This call can be completely removed, now that there are no skipped tests.

Reply:
True, I left it just because there's a separate PR to remove the skipping so I didn't want to trample over it. There's an identical call elsewhere in the file too